### ISSUE TITLE
Some Default Domain User Groups are empty after importing data 

### ISSUE THREAD

**Issue by ChoiSG** [helviojunior/knowsmore#2]
### Configuration  
impacket version:  0.10.0 
Python version:  3.10.9
Target OS:  Kali Linux 
Knowsmore version: 0.1.31
neo4j version: 4.4.16 
BloodHound GUI: 4.2.0

### Problem 
When importing bloodhound data to neo4j using knowsmore, there are some groups that are empty, without any member objects. For example, the default `domain controllers@<domain.com>` and `domain computers@<domain.com>` are empty. 

Compared to this, importing the data with the BloodHound GUI works normally and the 2 groups above have domain computers and controller objects inside the group. 

### Steps to Reproduce 
1. Start neo4j 4.4.16 and bloodhound GUI. 
```
neo4j console 
bloodhound 
```

2. Gather bloodhound data through an ingestor
```
python3 /opt/BloodHound.py/bloodhound.py -u Administrator -p 'Password123!' -d choi.local -c All,LoggedOn --zip
```

3. Import the data using Knowsmore v0.1.31. 
```
└─# python3 knowsmore.py --create-db -vv

KnowsMore v0.1.31 by Helvio Junior
Active Directory, BloodHound, NTDS hashes and Password Cracks correlation tool
https://github.com/helviojunior/knowsmore
    
 [+] Startup parameters
     command line: knowsmore --create-db -vv 
     verbosity level: 2
     module: create-db
     database file: /opt/knowsmore/knowsmore.db
  
 [+] start time 2023-01-31 22:54:12
 [+] Database created /opt/knowsmore/knowsmore.db
 [+] End time 2023-01-31 22:54:12

└─# python3 knowsmore.py --bloodhound --import-data 20230131224001_bloodhound.zip -vv 

KnowsMore v0.1.31 by Helvio Junior
Active Directory, BloodHound, NTDS hashes and Password Cracks correlation tool
https://github.com/helviojunior/knowsmore
    
 [+] Startup parameters
     command line: knowsmore --bloodhound --import-data 20230131224001_bloodhound.zip -vv 
     verbosity level: 2
     module: bloodhound
     operational mode: ImportMode.Import
     database file: /opt/knowsmore/knowsmore.db
  
 [+] start time 2023-01-31 22:54:45
 [?] BloodHound ZIP File identified, extracting...
 [?] checking file consistency...
 [?] 7 valid files in ZIP                         
 [?] importing domains...
 [!] Could not determine direction of trust... direction: Bidirectional
 [?] importing GPO...                             
 [?] importing OU...                              
 [?] importing groups...                          
 [?] calculating group chain...                                                  
 [?] importing computers...                       
 [?] loading groups from db...                                                  
 [?] importing users...                                                  
 [+] Imported objects                             
+-----+----------+------------+-----------+
|     | Type     |   Inserted |   Updated |
|-----+----------+------------+-----------|
|   1 | Computer |          4 |         0 |
|   2 | Domain   |          1 |         0 |
|   3 | GPO      |          3 |         0 |
|   4 | Group    |         52 |         0 |
|   5 | OU       |          1 |         0 |
|   6 | User     |         20 |         0 |
+-----+----------+------------+-----------+
 [+] End time 2023-01-31 22:54:45


└─# python3 knowsmore.py --bloodhound --sync-to 127.0.0.1:7687 -u neo4j -p bloodhound -d neo4j -vv 

KnowsMore v0.1.31 by Helvio Junior
Active Directory, BloodHound, NTDS hashes and Password Cracks correlation tool
https://github.com/helviojunior/knowsmore
    
 [+] Startup parameters
     command line: knowsmore --bloodhound --sync-to 127.0.0.1:7687 -u neo4j -p bloodhound -d neo4j -vv 
     verbosity level: 2
     module: bloodhound
     operational mode: ImportMode.Sync
     database file: /opt/knowsmore/knowsmore.db
  
 [+] start time 2023-01-31 22:55:04
 [?] Setting neo4j bloodhound schema...
 [?] Syncing objects...
 [?] Updating synced objects, wait a few seconds...
 [?] Syncing edges...                             
 [?] Updating synced objects, wait a few seconds...
 [?] Syncing owned objects...                     
 [+] End time 2023-01-31 22:55:05

```

4. Confirm that some default groups like `domain controllers@<domain.com>` and `domain computers@<domain.com>` are empty. 

![no-members](https://user-images.githubusercontent.com/36252854/215941899-b031f1ed-dd78-4a60-aff7-245045fedf91.png)

5. Confirm and validate that data imported via BloodHound GUI directly has correct members inside the groups. 

### Additional context  
Domain User groups like `domain admins@<domain.com>` contains correct domain users as members. It seems that this might be only a machine account-specific problem. 
Also, there might be more empty groups, I haven't looked through all groups.

**Comment by helviojunior** [comment]
@ChoiSG tks for reporting, i'm checking this issue.

**Comment by helviojunior** [comment]
@ChoiSG bugfix implemented and pushed at production in version v0.1.34
